build/devops.yml

```diff
@@ -0,0 +1,101 @@
+name: $(BuildID)_$(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+resources:
+- repo: self
+  clean: true
+phases:
+
+- phase: windowsx86
+  displayName: 'Build Windows x86'
+  queue:
+    name: Hosted VS2017
+    demands:
+    - Cmd
+    - msbuild
+    - visualstudio
+
+  steps:
+  - task: BatchScript@1
+    displayName: 'Setup VS vars'
+    inputs:
+      filename: '"C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsDevCmd.bat"'
+      modifyEnvironment: true
+
+  - task: CMake@1
+    displayName: CMake
+    inputs:
+      cmakeArgs: '.. -Drun_int_tests:bool=ON -G "Visual Studio 15 2017"'
```

Comment:
>cmakeArgs [](start = 6, length = 9)

maybe args should explicitly list "Debug" as the config.

Reply:
This phase builds RelWithDebInfo. Do you mean to add RelWithDebInfo to the displayName?

---
In reply to: [270525694](https://github.com/Azure/azure-macro-utils-c/pull/7#discussion_r270525694) [](ancestors = 270525694)

Reply:
No action is further needed, reviewer was confused!!!!

---
In reply to: [270540939](https://github.com/Azure/azure-macro-utils-c/pull/7#discussion_r270540939) [](ancestors = 270540939,270525694)